ASWF/DCO.md

```diff
@@ -0,0 +1 @@
+https://developercertificate.org/
```

Comment:
You could also link to https://github.com/AcademySoftwareFoundation/tac/blob/master/process/contributing.md#contribution-sign-off

Reply:
You mean from the CONTRIBUTING, and not have the stub DCO.md file at all?

Reply:
That works too - just saw this was in here and was trying to understand the intention.

Reply:
The intention was to copy verbatim what I saw in OpenColorIO.

Reply:
I'm just going to eliminate the DCO file, leave the instructions I added to CONTRIBUTING.